main.py

```diff
@@ -225,14 +256,16 @@ def run_search(kontaktdaten, check_delay):
         print(
             f"Kontaktdaten wurden geladen für: {kontakt['vorname']} {kontakt['nachname']}\n")
 
+        notifications = kontaktdaten["notifications"]
```

Comment:
Mit `--read-only` (bzw. `-r`) existiert dieser Key nicht zwingend. Ich denke wir sollten ihn auch nicht ab jetzt plötzlich obligatorisch machen, sondern lieber optional. Heißt:

```suggestion
        notifications = kontaktdaten.get("notifications", {})
```